doc/notebooks/example_notebook.ipynb

```diff
@@ -10,9 +10,13 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": 3
+   "version": "3.8.3"
   },
-  "orig_nbformat": 2
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python383jvsc74a57bd07e402764773cbeaa2d2d6f3815f29bb5da9599569c66a0a6d6a02c360a55847f",
+   "display_name": "Python 3.8.3 64-bit ('base': conda)"
+  }
```

Comment:
You can manually edit the file to revert this change. Something like `git checkout master -- doc/notebooks/example_notebook.ipynb` should bring the old state too if you want to go the Git way, but commit all your changes before, I'm wring the command without checking it.

Reply:
I think I just need to delete this file completely since I'm working on it in the binder-button PR.

Reply:
Making it same as on the master branch is what "removes" it from the PR. Deleting in PR is really deleting.